chromium_src/chrome/browser/policy/configuration_policy_handler_list_factory.cc

```diff
@@ -0,0 +1,52 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "base/stl_util.h"
+#include "brave/browser/tor/buildflags.h"
```

Comment:
is there a dep for this in the browser target we add to chrome/browser?

Reply:
Yes.
```
simon-home:src simonhong$ gn path ./out/Debug/ chrome/browser  brave/browser/tor:buildflags
//chrome/browser:browser --[private]-->
//brave/browser:browser_process --[private]-->
//brave/browser/tor:tor --[public]-->
//brave/browser/tor:buildflags
```

Reply:
same as above, we should add a dep for `//brave/browser/tor:buildflags` directly to `//brave/browser:browser_process`
It actually seems quite weird that tor would have a dep on buildflags because tor should only be included if the buildflag is set

Reply:
Done.
```
//chrome/browser:browser --[private]-->
//brave/browser:browser_process --[private]-->
//brave/browser/tor:buildflags
```
And we added `tor` into two places. One is unconditioned deps list and the other is in `enable_tor`. So, `tor` in unconditioned deps list was removed.